jiant/utils/serialize.py

```diff
@@ -57,7 +58,48 @@ def bytes_to_float(b):
     return float(crc32(b) & 0xFFFFFFFF) / 2 ** 32
 
 
-def read_records(filename, repeatable=False, fraction=None):
+def read_records(filename, repeatable=False, fraction=1.0):
+    """Streaming read records from file.
+
+    Args:
+      filename: path to file of b64-encoded pickles, one per line
+      repeatable: if true, returns a RepeatableIterator that can read the file
+        multiple times.
+      fraction: if set to a float between 0 and 1, load only the specified percentage
+        of examples. Hashing is used to ensure that the same examples are loaded each
+        epoch.
+
+    Returns:
+      iterable, possible repeatable, yielding deserialized Python objects
+    """
+
+    if fraction < 1.0:
+        frac_filename = f"{filename}__fraction_{fraction}"
```

Comment:
Because with this change the fraction argument is going to generate new (possibly big) files, we should consider giving the user a heads up. Can you add a note to the related options in the config mentioning this new behavior?